### PR TITLE
add felevnyito-es-felveteli-vacsora-2022

### PR DESCRIPTION
**Pull request by SepsiLaszlo**

### CODE REVIEW

File: src/content/posts/2022-11-08-felevnyito-es-felveteli-vacsora---2022.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Ez a félév sem indulhatott el egy remek simonyis Félévnyitó és Felvételi Vacsora nélkül. A nyár után már nagyon vártuk, hogy találkozhassunk a többi szakkollégiumi kör tagjaival. A rendezvény elején elfoglaltuk a szokásos asztalunkat, hogy legyen egy stabil kiindulópontja az estének.
+
+![f&f-2021-table](https://warp.sch.bme.hu/images/f-f-2021-table)
+
+## Új tagokkal bővültünk
+
+Az est középpontjában természetesen a felvételizők álltak. Idén 3 felvételizőnk vágott bele a felvételi folyamatba. Nagyon meggyőző teljesítményeket nyújtottak ebben az évben a felvételizői nk, mindannyian sikeresen vették az akadályokat. A körvezetőnk büszkén adta át nekik a körös és szakkollégiumi tagságot jelképező oklevelet.
+
+![f&f-2021-table](https://warp.sch.bme.hu/images/f-f-2021-okelvel
```
Comment (berenteb) [inline src/content/posts/2022-11-08-felevnyito-es-felveteli-vacsora---2022.md:31]: Woops new line

File: src/content/posts/2022-11-08-felevnyito-es-felveteli-vacsora---2022.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+![f&f-2021-cover](https://warp.sch.bme.hu/images/f-f-2021-cover)
+
+Ez a félév sem indulhatott el egy remek simonyis Félévnyitó és Felvételi Vacsora nélkül. A nyár után már nagyon vártuk, hogy találkozhassunk a többi szakkollégiumi kör tagjaival. A rendezvény elején elfoglaltuk a szokásos asztalunkat, hogy legyen egy stabil kiindulópontja az estének.
+
+![f&f-2021-table](https://warp.sch.bme.hu/images/f-f-2021-table)
+
+## Új tagokkal bővültünk
+
+Az est középpontjában természetesen a felvételizők álltak. Idén 3 felvételizőnk vágott bele a felvételi folyamatba. Nagyon meggyőző teljesítményeket nyújtottak ebben az évben a felvételizői nk, mindannyian sikeresen vették az akadályokat. A körvezetőnk büszkén adta át nekik a körös és szakkollégiumi tagságot jelképező oklevelet.
```
Comment (berenteb) [inline src/content/posts/2022-11-08-felevnyito-es-felveteli-vacsora---2022.md:29]: Accidental whitespace at the end

**Review by berenteb — COMMENTED** [review]
Some small things but seems ok

**Review by DannySS123 — COMMENTED** [review]
Looks awesome! 💯 🔥 
Just correct those small things that Bálint found.

**Comment by SepsiLaszlo** [comment]
Thank you for finding the typos! I made the suggested corrections.

**Pull request merged**